[MLOB-2597] Add LangGraph LLM Observability Auto-Instrumentation Documentation (#28669)
DataDog/documentation: content/en/llm_observability/setup/auto_instrumentation.md

| [Anthropic](#anthropic)                    | >= 0.28.0          | >= 2.10.0         |
| [Google Gemini](#google-gemini)            | >= 0.7.2           | >= 2.14.0         |
| [Vertex AI](#vertex-ai)                    | >= 1.71.1          | >= 2.18.0         |

> | [LangGraph](#langgraph)                    | >= 0.2.23          | >= 3.5.0          |

Author: ```suggestion
| [Langgraph](#langgraph)                    | >= 0.2.23          | >= 3.5.0          |
```

> | [Langgraph](#langgraph)                    | >= 0.2.23          | >= 3.5.0          |

Reviewer: ```suggestion
| [LangGraph](#langgraph)                    | >= 0.2.23          | >= 3.5.0          |
```

  - `chat.send_message()`
  - `chat.send_message_async()`

> ## Langgraph

Reviewer: ```suggestion
## LangGraph
```

> The Langgraph integration automatically traces `Pregel/CompiledGraph` and `RunnableSeq (node)` invocations made through the [Langgraph Python SDK][33].

Reviewer: ```suggestion
The LangGraph integration automatically traces `Pregel/CompiledGraph` and `RunnableSeq (node)` invocations made through the [LangGraph Python SDK][33].
```

### Traced methods

> The Langgraph integration instruments synchronous and asynchronous versions of the following methods:

Reviewer: ```suggestion
The LangGraph integration instruments synchronous and asynchronous versions of the following methods:
```

  - `chat.send_message()`
  - `chat.send_message_async()`

> ## LangGraph

Author: ```suggestion
## Langgraph
```

> The LangGraph integration automatically traces `compiled graph` and `node` invocations made through the [LangGraph Python SDK][26].

Author: ```suggestion
The Langgraph integration automatically traces `compiled graph` and `node` invocations made through the [LangGraph Python SDK][26].
```

> The Langgraph integration automatically traces `compiled graph` and `node` invocations made through the [LangGraph Python SDK][26].

Author: ```suggestion
The Langgraph integration automatically traces `compiled graph` and `node` invocations made through the [Langgraph Python SDK][26].
```

> The Langgraph integration automatically traces `compiled graph` and `node` invocations made through the [Langgraph Python SDK][26].

Reviewer 2: ```suggestion
The Langgraph integration automatically traces `Pregel/CompiledGraph` and `RunnableSeq (node)` invocations made through the [Langgraph Python SDK][26].
```

### Traced methods

> The LangGraph integration instruments the following methods:

Author: ```suggestion
The Langgraph integration instruments the following methods:
```

> The Langgraph integration instruments the following methods:

Reviewer 2: ```suggestion
The Langgraph integration instruments synchronous and asynchronous versions of the following methods:
```

> - [Compiled graphs][27]
> - [Nodes][28]

Reviewer 2: ```suggestion
- [CompiledGraph.invoke(), Pregel.invoke(), CompiledGraph.stream(), Pregel.stream()][27]
- [RunnableSeq.invoke()][28]
```
